### ISSUE TITLE
On first run - got an error on opening an issue

### ISSUE THREAD

**Issue by bbracha-evinced** [github/accessibility-scanner#44]
Seems that the scanner found some a11y issues, but it fails, to my understanding, trying to create an issue.
I don't understand why it refers to the docs website.

The error I see:
Failed on filing: [object Object] HttpError: Not Found - https://docs.github.com/rest/issues/issues#create-an-issue

<img width="1486" height="627" alt="Image" src="https://github.com/user-attachments/assets/30f5ff38-d7a5-42a2-ad3c-90301411bde8" />

**Comment by nwanduka** [comment]
I'm also getting this error. I've followed the fix suggested but the error persists. Here's [the link to the workflow run logs](https://github.com/nwanduka/chaoss-website/commit/3de79d3a5ac8509304332e7a4c42efc15569bef5/checks/58812253340/logs). Any help?
